extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperimentInitArticleEntry.js

```diff
@@ -14,9 +14,9 @@ require(['jquery', 'mw', 'VisualEditorTourExperimentInit', 'ext.wikia.spitfires.
 				$editButton.popover({
 					content: '<div class="ve-tour-experiment">' +
 						'<div class="close"></div>' +
-						'<h2>Enter edit mode</h2>' +
-						'<p>Click on the Edit button to go into edit mode - there you will be able to add content ' +
-						'and media to an article page.</p>' +
+						'<h2>編集モードを開く</h2>' +
+						'<p>「編集」ボタンをクリックすると編集モードを開始し、記事が掲載されているページにコンテンツやメディアを追加する'+
+						'ことができます。</p>' +
```

Comment:
For the strings that are encoded in the config JS file, you use  mw.config.get('wgUserLanguage'); to decide what language to serve to the user.

If that language is set to 'en', this text will still end up in Japanese.
You may want to add a test for  mw.config.get('wgUserLanguage') === 'ja' and only then display the Japanese here.